pkg/kubelet/config/file_linux_test.go

```diff
@@ -370,6 +386,10 @@ func expectUpdate(t *testing.T, ch chan interface{}, testCase *testCase) {
 		select {
 		case got := <-ch:
 			update := got.(kubetypes.PodUpdate)
+			if len(update.Pods) == 0 {
```

Comment:
Was there a specific reason they weren't being filtered before or is this just fixing a bug in the tests?